console/console.go

```diff
@@ -305,14 +320,22 @@ func (c *Console) Welcome() {
 		console.log(" datadir: " + admin.datadir);
 	`)
 	// List all the supported modules for the user to call
-	if apis, err := c.client.SupportedModules(); err == nil {
-		modules := make([]string, 0, len(apis))
-		for api, version := range apis {
-			modules = append(modules, fmt.Sprintf("%s:%s", api, version))
+	apis, err := c.client.SupportedModules()
+	if err != nil {
+		if !isMethodNotFoundError(err) {
+			fmt.Fprintln(c.printer)
+			return
 		}
-		sort.Strings(modules)
-		fmt.Fprintln(c.printer, " modules:", strings.Join(modules, " "))
+		apis = apisAll
+		fmt.Fprintln(c.printer, `WARNING: use all modules because "rpc_moduels" does not supported`)
+		fmt.Fprintln(c.printer, ` API calls associated with specific namespaces may fail if a service node does not open those namespaces.`)
```

Comment:
@zacscoding I understood that you enables all the apis of the console because the server did not give the supporting api console information.
However, this explanation is quite confusing to me.. what about below?
```
fmt.Fprintln(c.printer, `WARNING: Supporting module information is unknown since "rpc_modules" api is not supported`)
fmt.Fprintln(c.printer, `Since the console doesn't know the available modules, the console made it possible to call all modules but it fails if it's unsupported`)
```